features/rule-mgt/org.wso2.carbon.identity.rule.management.server.feature/resources/identity/rulemeta/fields.json

```diff
@@ -58,5 +58,45 @@
         }
       ]
     }
+  },
+  "flow": {
+    "field": {
+      "name": "flow",
+      "displayName": "flow"
+    },
+    "operators": [
+      "equals",
+      "notEquals"
+    ],
+    "value": {
+      "inputType": "options",
+      "valueType": "string",
+      "values": [
```

Comment:
In fields.json let's not define these values.
Let's override them at flows.json

Reply:
There is a validation in the `OptionInputValue` class on the values attribute.
https://github.com/wso2/carbon-identity-framework/blob/db7185f0b160a243ce4ff4bf0b0732f0836aad49/components/rule-mgt/org.wso2.carbon.identity.rule.metadata/src/main/java/org/wso2/carbon/identity/rule/metadata/model/OptionsInputValue.java#L54

If we are to go with empty array, we have to remove that validation

Reply:
Let's update to accept an empty array. And add the validation at the static rule loader to check if it has not overridden